tests/unit/conftest.py

```diff
@@ -18,6 +18,7 @@
 """
 
 import asyncio
+import concurrent.futures
```

Comment:
Shouldn't this be from concurrent import futures?  Or is that too confusing?

Reply:
I have no strong preference and in this case I probably just copy/pasta'd it from an example on how to test futures. I don't mind more verbose/descriptive unless its a pain in the butt (multiple uses, causing lines to wrap), but I can update if you want.

Reply:
Well, the style guide says 'from', so I guess unless it's too confusing...

Reply:
To be clear the style guide says from is preferred but that both are ok.

Reply:
Er...  I don't think that's the intended interpretation of the guide.  Unless I'm looking in the wrong place or something.

Reply:
I was looking at https://github.com/google/styleguide/blob/gh-pages/pyguide.md#233-decision

Reply:
Huh.  Interesting.  I was looking at 2.2.